cloudify_aws/eks/resources/cluster.py

```diff
@@ -221,6 +264,28 @@ def create(ctx, iface, resource_config, **_):
         except TypeError as error:
             raise NonRecoverableError(
                 'kubeconf not json serializable {0}'.format(text_type(error)))
+    region_name = ctx.node.properties['client_config']['region_name']
+    aws_resource_arn = ctx.instance.runtime_properties['aws_resource_arn']
+    fargate = iface.is_fargate()
+    node_group = iface.is_nodegroup()
+    if fargate and node_group:
+        node_type = 'fargate-nodegroup'
+    elif fargate:
+        node_type = 'fargate'
+    else:
+        node_type = 'nodegroup'
+    zones = get_zones(
+        ctx,
+        resource_config['resourcesVpcConfig']['subnetIds'])
+    utils.add_new_labels(
```

Comment:
This means users with Cloudify <5.2 cant use this plugin because there are no labels.